README.md

```diff
@@ -1 +1,18 @@
-## Canvas Course Manager
\ No newline at end of file
+## Canvas Course Manager
```

Comment:
This seems straight forward, lot of complexities are lifted by NEXT.JS framework it self.